Sources/Features/DappsAndPersonas/PersonaDetails/PersonaDetails.swift

```diff
@@ -62,6 +62,7 @@ public struct PersonaDetails: Sendable, FeatureReducer {
 
 	public enum ViewAction: Sendable, Equatable {
 		case appeared
+		case accountTapped(AccountAddress)
```

Comment:
hmm this view action is added, but... we are never emitting it? since `PersonaDatails+View` has not been edited?

Reply:
bad merge, was removed in the meantime in main.